RTClib.h

```diff
@@ -444,17 +261,26 @@ class RTC_PCF8563 {
    use. NOTE: this is immune to millis() rollover events.
 */
 /**************************************************************************/
-class RTC_Millis {
+class RTC_Millis : RTC {
 public:
+  boolean begin(const DateTime &dt = DateTime::COMPILE_DT);
   /*!
-      @brief  Start the RTC
-      @param dt DateTime object with the date/time to set
+    @brief  Simulate if the RTC is running
+    @return true
   */
-  static void begin(const DateTime &dt) { adjust(dt); }
-  static void adjust(const DateTime &dt);
-  static DateTime now();
+  boolean isrunning(void) { return true; }
+  /*!
+    @brief Simulate if the RTC has lost power
+    @return false
+  */
+  boolean lostPower(void) { return false; }
+  void adjust(const DateTime &dt);
+  void adjustDrift(const int drift);
+  DateTime now();
 
 protected:
+  static uint32_t millisPerSecond; ///< Number of milliseconds reported by
+                                   ///< millis() per "true" (calibrated) second
```

Comment:
This should be `const`.

Reply:
This variable is used in the `adjustDrift()` and `now()` functions of the `RTC_Millis` in the same way that the `microsPerSecond` variable is used by the `RTC_Micros` class. I've now updated it so that `RTC_Millis` has it's own `microsPerSecond` variable so that the `adjustDrift()` function always takes a `ppm` parameter.